LoggingKata/TacoParser.cs

```diff
@@ -6,13 +6,37 @@
     public class TacoParser
     {
         readonly ILog logger = new TacoLogger();
-        
+
         public ITrackable Parse(string line)
         {
-            logger.LogInfo("Begin parsing");
+            var cells = line.Split(',');
+            if (cells.Length < 3 || cells.Length > 3)
+            {
+                return null;
+            }
+                logger.LogInfo("Begin parsing");
+
+            TacoBell newTacoBell = new TacoBell();
+            {
```

Comment:
This opening curly (and the scope it defines) isn't necessary. It looks like it was intended to be object initializer syntax, but the semicolon at the end of `TacoBell newTacoBell = new TacoBell();` is preventing that from being the case